package.json

```diff
@@ -2,17 +2,40 @@
   "name": "atlas",
   "version": "0.1.0",
   "private": true,
+  "type": "module",
+  "scripts": {
+    "start": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "lint": "eslint '*/**/*.{js,ts,tsx}' --color",
+    "lint:fix": "eslint '*/**/*.{js,ts,tsx}' --fix"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
   "dependencies": {
-    "@craco/craco": "^6.4.3",
     "@emotion/react": "^11.8.1",
     "@emotion/styled": "^11.8.1",
     "@mui/base": "^5.0.0-alpha.80",
     "@mui/icons-material": "^5.4.4",
     "@mui/material": "^5.4.4",
     "@popperjs/core": "^2.10.2",
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
```

Comment:
Please, let both of these dependency here, these will be useful to some cases test